Fix broken link in Challenge_creation page

kurianbenoy opened the pull request:
Fix #2302 
- [X] Change link to https://github.com/Cloud-CV/EvalAI-Starters

kurianbenoy commented:
Hello

RishabhJain2018 commented:
@kurianbenoy if possible can you please resolve the conflicts here or open up a new PR.

kurianbenoy commented:
OK will resolve soon Rishab

On Fri, 12 Jul 2019, 5:13 p.m. Rishabh Jain, <notifications@github.com>
wrote:

> @kurianbenoy <https://github.com/kurianbenoy> if possible can you please
> resolve the conflicts here or open up a new PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Cloud-CV/EvalAI/pull/2304?email_source=notifications&email_token=AF3UDJW2TXM7WXJ6I2IP3MTP7BU63A5CNFSM4HKT5452YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZZQW4A#issuecomment-510856048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF3UDJU3OCOBH55CHNBBJ43P7BU63ANCNFSM4HKT545Q>
> .
>

kurianbenoy commented:
@RishabhJain2018 the file is now deleted and I tried searching for a similar file  with contents using GitHub search, Yet I couldn't find any references for the same

deshraj commented:
@kurianbenoy: Sorry, we recently revamped our documentation and we had to restructure a lot of things. In the process of restructuring, we replaced the file with `host_challenge.md` and fixed all/most the issues that might be there. Please feel free to take a look at current documentation and send fixes as PR. 

We will try to make sure that the PRs get merged as soon as possible before it gets marked as stale. Thanks for all your contributions. We really appreciate your time and effort. :)

kurianbenoy commented:
> @kurianbenoy: Sorry, we recently revamped our documentation and we had to restructure a lot of things. In the process of restructuring, we replaced the file with `host_challenge.md` and fixed all/most the issues that might be there. Please feel free to take a look at current documentation and send fixes as PR.

> We will try to make sure that the PRs get merged as soon as possible before it gets marked as stale. Thanks for all your contributions. We really appreciate your time and effort. :)
Thanks for your kind words @deshraj, just saw this now :+1. I hope I continue contributing to EvalAI and CloudCV in the coming months as well
